src/webui/www/private/addpeers.html

```diff
@@ -46,10 +46,10 @@
                         hashes: hash,
                         peers: peers.join("|")
                     },
-                    onFailure: function() {
+                    onFailure: () => {
```

Comment:
I have a bad feeling about that. Because JS allows even simpler syntax for propery functions.
```suggestion
                    onFailure() {
```

Reply:
>Because JS allows even simpler syntax for declaring object methods.

Although simpler, the result function is not a arrow function but a plain function.

Reply:
> Although simpler, the result function is not a arrow function but a plain function.

I wonder if the difference is not only in syntax?

Reply:
>I wonder if the difference is not only in syntax?

yes, arrow function is different to plain function.

Reply:
> Although simpler, the result function is not an arrow function but a plain function.

But you change them from regular functions anyway. Which means there should be no difference. Because in other case they would explode.

> And I don't think it looks nice when mixed with other declarations.

Ok.

> I wonder if the difference is not only in syntax?

Yes. Regular and arrow functions have different notion of `this`.

Reply:
>But you change them from regular functions anyway. Which means there should be no difference. Because in other case they would explode.

Yes. I also wanted to unify the coding style and have some tool to check/enforce it.